tests/test_transport_network.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+
+import pathlib
+
+import pytest  # noqa: F401
+
+import r5py
+import com.conveyal.r5
+import java.time
+
+# test data sets
+DATA_DIRECTORY = pathlib.Path(__file__).absolute().parent.parent / "docs" / "data"
+OSM_PBF = DATA_DIRECTORY / "kantakaupunki.osm.pbf"
+GTFS = DATA_DIRECTORY / "GTFS.zip"
+
+
+class Test_TransportNetwork:
+    @pytest.fixture
+    def gtfs_timezone(self):
+        yield "Europe/Helsinki"
+
+    @pytest.fixture(scope="session")
+    def transport_network_from_test_files(self):
+        transport_network = r5py.TransportNetwork(OSM_PBF, [GTFS])
+        yield transport_network
+
+    @pytest.fixture(scope="session")
+    def transport_network_from_test_directory(self):
+        yield r5py.TransportNetwork.from_directory(DATA_DIRECTORY)
+
+    def test_init_from_files_and_dir_cover_same_extent(
+        self, transport_network_from_test_files, transport_network_from_test_directory
+    ):
+        assert (
+            transport_network_from_test_files._transport_network.getEnvelope()
+            == transport_network_from_test_directory._transport_network.getEnvelope()
+        )
+
+    @pytest.mark.parametrize(
+        ["transport_network"],
+        [
+            (pytest.lazy_fixture("transport_network_from_test_files"),),
```

Comment:
Question to educate myself as haven't seen these before 🙂 What are these lazy fixtures? What do they do?